### DISCUSSION TITLE
TableGroupRow: Recommended alternative

### DISCUSSION THREAD

**Discussion by tkeiner** [UI5/webcomponents-react#6462] [General]
Hello everyone,
With version 2.0.0 the TableGroupRow component was removed. I have found no explanation for this and no recommendation for an alternative component or solution. I find this confusing, especially as the standard web components still have this component. https://sap.github.io/ui5-webcomponents/components/compat/TableGroupRow/

Could you please clarify this issue?

Kind regards, Tobias

**Comment by Lukas742** [comment]
Hi @tkeiner 
what you're referring to is the Table in the `compat` package of ui5-webcomponents. You can use the [old table](https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/legacy-components-table--docs) with our wrapper by using our compat package as well. 
If you want grouping to be implemented in the [new table](https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/data-display-table--docs)([ui5wc docs](https://sap.github.io/ui5-webcomponents/components/Table/)), then please open a [feature request](https://github.com/SAP/ui5-webcomponents/issues/new/choose) in the ui5-webcomponents repo.
